sql/moz-fx-data-shared-prod/search_derived/mobile_search_aggregates_v1/query.sql

```diff
@@ -21,6 +21,7 @@ SELECT
   SUM(ad_click_organic) AS ad_click_organic,
   SUM(search_with_ads) AS search_with_ads,
   SUM(unknown) AS unknown,
+  NULL AS normalized_engine,
```

Comment:
It might be fine without it, but probably a good idea to

```suggestion
  CAST(NULL AS STRING) AS normalized_engine,
```